### DISCUSSION TITLE
Error running custom Docker action

### DISCUSSION THREAD

**Discussion by unguiculus** [community/community#26136] [Actions]
<p>I can successfully run an action using <a href="https://github.com/actions/docker/" rel="nofollow noopener">GitHub Actions for Docker</a>. However, when I create my own Docker image for an action that runs Docker, execution fails without an obvious reason. It first shows “Cancelled!” and then turns into “Failed!”. I basically just copied the Dockerfile, made minor adjustments and used a custom entrypoint script. Am I missing something?</p>
<p>Here’s the action run:</p>
<p><a href="https://github.com/unguiculus/actions-test/runs/47936410" rel="nofollow noopener">https://github.com/unguiculus/actions-test/runs/47932633</a></p>
<p>Thanks,</p>
<p>Reinhard</p>

**Comment by judoole** [comment]
<p>I’m uncertain as to why, but you could try to add a **CMD [""] **to your Dockerfile.</p>

**Comment by unguiculus** [comment, marked as answer]
<p>Thanks, I found the problem. The entrypoint script was not executable. Unfortunately, no error was shown.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/unguiculus">@unguiculus</a>, thanks for coming back and sharing what worked for you! We’re looking forward to seeing you around!</p>
